packages/express/src/generators/application/application.ts

```diff
@@ -60,13 +60,8 @@ server.on('error', console.error);
     toJS(tree);
   }
 }
-// TODO (nicholas): Remove After Nx 16
-// @deprecated Use `nx g @nrwl/node:app --framework=express instead.
-export async function applicationGenerator(tree: Tree, schema: Schema) {
-  console.warn(
-    'As of Nx 16 using `nx g @nrwl/express:app` has been deprecated! Use `nx g @nrwl/node:app --framework=express instead.'
-  );
 
+export async function applicationGenerator(tree: Tree, schema: Schema) {
```

Comment:
I think we either remove this or reuse it in `@nrwl/node:app` when choosing express. Right now it is duplicated.

Out of scope for this PR, but should do a follow-up.

Reply:
I think we should reuse it.

I think `@nx/node` can do the framework.. but it would proxy to `@nx/express` (which is the real one).

I believe it's similar for `@nx/node:app --framework nest` right?